spec/koans.js

```diff
@@ -1,601 +1,656 @@
-      //expect(evaluated).toBe(x + '+' + y);
-    });
-
-  });
-
-  describe('can evaluate any expression, wrapped inside "${...}"', function() {
-
-    it('all inside "${...}" gets evaluated', function() {
-      var evaluated = Number(`x+y`);
-      //expect(evaluated).toBe(x+y);
-    });
-
-    it('inside "${...}" can also be a function call', function() {
-      function getSchool(){
-        return 'Ironhack';
+      let str = `like a string`
+      expect(str).toEqual('like a string')
+    })
+  })
+
+  let x = 42
+  let y = 23
+
+  describe('can evaluate variables, which are wrapped in "${" and "}"', function () {
+    it('e.g. a simple variable "${x}" just gets evaluated', function () {
+      let evaluated = `x=${x}`
+      expect(evaluated).toBe('x=' + x)
+    })
+
+    it('multiple variables get evaluated too', function () {
+      var evaluated = `${x}+${y}`
+      expect(evaluated).toBe(x + '+' + y)
+    })
+  })
+
+  describe('can evaluate any expression, wrapped inside "${...}"', function () {
+    it('all inside "${...}" gets evaluated', function () {
+      var evaluated = Number(`${x + y}`)
+      expect(evaluated).toBe(x + y)
+    })
+
+    it('inside "${...}" can also be a function call', function () {
+      function getSchool() {
+        return 'Ironhack'
       }
-      var evaluated = `getSchool()`;
-      //expect(evaluated).toBe('Ironhack');
-    });
-
-  });
-
-});
+      var evaluated = `${getSchool()}`
+      expect(evaluated).toBe('Ironhack')
+    })
+  })
+})
 
 describe('The object literal allows for new shorthands. ', () => {
-
-  const x = 1;
-  const y = 2;
+  const x = 1
+  const y = 2
 
   describe('with variables', () => {
     it('the short version for `{y: y}` is {y}', () => {
       /*.....*/
-      //expect(short).toEqual({y: y});
-    });
+      short = { y }
+      expect(short).toEqual({ y: y })
+    })
     it('works with multiple variables too', () => {
       /*.....*/
-      //expect(short).toEqual({x: x, y: y});
-    });
-  });
+      short = { x, y }
+      expect(short).toEqual({ x: x, y: y })
+    })
+  })
 
   describe('with methods', () => {
-
-    const func = () => func;
+    const func = () => func
 
     it('using the name only uses it as key', () => {
       /*.......*/
-      //expect(short).toEqual({func: func});
-    });
+      short = { func }
+      expect(short).toEqual({ func: func })
+    })
 
     it('a different key must be given explicitly, just like before ES6', () => {
       /*.......*/
-      //expect(short).toEqual({otherKey: func});
-    });
-  });
-
-});
+      short = { otherKey: func }
+      expect(short).toEqual({ otherKey: func })
+    })
+  })
+})
 
 describe('destructuring arrays makes shorter code. ', () => {
-
   it('extract value from array, e.g. extract 0 into x like so `let [x] = [0];`', () => {
-    let firstValue = [1];
-    //expect(firstValue).toEqual(1);
-  });
+    let firstValue = [1]
+    let [x] = firstValue
+    firstValue = x
```

Comment:
```suggestion
    expect(x).toEqual(1)
```

Te bastaba poner la x para que funcione 😄